pkg/quota/evaluator/core/pods.go

```diff
@@ -172,16 +172,7 @@ func PodMatchesScopeFunc(scope api.ResourceQuotaScope, object runtime.Object) bo
 }
 
 func isBestEffort(pod *api.Pod) bool {
-	// TODO: when we have request/limits on a pod scope, we need to revisit this
-	for _, container := range pod.Spec.Containers {
-		qosPerResource := util.GetQoS(&container)
-		for _, qos := range qosPerResource {
-			if util.BestEffort == qos {
-				return true
-			}
-		}
-	}
-	return false
+	return util.GetPodQos(pod) == util.BestEffort
```

Comment:
https://github.com/kubernetes/kubernetes.github.io/pull/622

This is an advanced feature of resourcequota.  See step 5 in PR above that discusses scopes filtering the set of resources a quota tracks in an example.

Full doc on scopes and quota:
https://github.com/kubernetes/kubernetes.github.io/pull/620